Sources/StatefulTabView/Helpers/Tab.swift

```diff
@@ -26,6 +27,19 @@ public struct Tab {
         self.view = AnyView(content())
     }
     
+    public init<T>(title: String,
+                   selectedImageName: String,
+                   unselectedImageName: String,
```

Comment:
I like these initializers but I think it would be easier to combine the two and just have `unselectedImageName` be passed in like badge value where it is not required. So basically `unselectedImageName: String? = nil`. This way we don't need two initializers for each image type. All that would be needed is that the bar item would need to be set differently based on if there was or was not an unselected image name.

Reply:
Totally Agree